libsodium-sys/src/crypto_generichash.rs

```diff
@@ -70,3 +94,55 @@ fn test_crypto_generichash_primitive() {
         assert_eq!(s, crypto_generichash_PRIMITIVE.as_bytes());
     }
 }
+
+#[test]
+fn test_crypto_generichash_statebytes() {
+    assert!(unsafe { crypto_generichash_statebytes() } > 0);
```

Comment:
Don't test that it's greater than 0, test that it's equal to the known constant.